common/test/acceptance/tests/helpers.py

```diff
@@ -350,6 +353,33 @@ def is_404_page(browser):
     return 'Page not found (404)' in browser.find_element_by_tag_name('h1').text
 
 
+def create_multiple_choice_problem(problem_name):
+    """
+    Return the Multiple Choice Problem Descriptor, given the name of the problem.
+    """
+    factory = MultipleChoiceResponseXMLFactory()
+    xml_data = factory.build_xml(
+        question_text='The correct answer is Choice 2',
+        choices=[False, False, True, False],
+        choice_names=['choice_0', 'choice_1', 'choice_2', 'choice_3']
+    )
+
+    return XBlockFixtureDesc(
+        'problem',
+        problem_name,
+        data=xml_data,
+        metadata={'rerandomize': 'always'}
```

Comment:
Question: Should we set the problem weight to a non-default value?  Maybe a weight of 2 or 10, for example?